### ISSUE TITLE
File Layout error in the Ryujinx download

### ISSUE THREAD

**Issue by BedeScarlet** [HDR-Development/HewDraw-Remix#284]
In the Ryujinx download, the layout for the skyline plugin is as follows:
`mods/contents/01006a800016e000/skyline/romfs/skyline/plugins/libhdr.nro`
The proper file path is:
`mods/contents/01006a800016e000/romfs/skyline/plugins/libhdr.nro`
When dragging and dropping the download contents into the Ryujinx directory, you have to manually go and put the /romfs/ folder in its correct place. Not a big issue, but annoying for lazy people (me).

**Comment by zandm7** [comment]
Would fixing this remove the prerequisite to manually install Skyline before HDR will work (on a fresh install)?

**Comment by blu-dev** [comment]
I don't believe this is actually an issue, as there are numerous reports from people (including myself) who haven't had any issue.

Not sure if there is a Ryujinx-wide setting that you can change, but from fresh install this current packaging is fine.

> Would fixing this remove the prerequisite to manually install Skyline before HDR will work (on a fresh install)?

No

**Comment by zandm7** [comment]
OK so I took a closer look at this out of curiosity.

It seems like @WannaBeDev64 is advocating for the usage of a more standardized and simpler plugin file structure.

I tried moving all of the HDR .nro plugins from `/skyline/romfs...` to `/romfs...` and the mod still works + Ryujinx seems to prioritize loading the plugins located in `/romfs...` over the ones contained in the default file structure.

The functionality of the mod doesn't seem to be affected either way, so this is probably a very low priority issue (unless I'm missing something) but it seems like potentially a good optimization to do at some point.
